tests/unit/test_itrf.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+
+from iceflow.itrf import check_itrf
+
+
+def test_check_itrf():
+    assert check_itrf("Not an ITRF string") is False
+    assert check_itrf("ITRF") is False
```

Comment:
No preference, just thinking out loud. What about `assert not check_itrf("bad itrf")` and `assert check_itrf("good itrf")`?

Reply:
ad67cae